release-notes/opensearch-release-notes-2.8.0.md

```diff
@@ -0,0 +1,490 @@
+Release Notes 2.8.0
```

Comment:
Add links to OpenSearch and OpenSearch Dashboards release notes.

Reply:
Added now.